### PR TITLE
fix: dev: #206: Remove ClusterManager dependency from CachingPrestoNativeAzureFileSystem

### PR DESCRIPTION
**Pull request by abhishekdas99**

### CODE REVIEW

File: rubix-presto/src/main/java/com/qubole/rubix/presto/CachingPrestoS3FileSystem.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public CachingPrestoS3FileSystem()
   public void initialize(URI uri, Configuration conf) throws IOException
   {
     try {
-      initializeClusterManager(conf, ClusterType.HADOOP2_CLUSTER_MANAGER);
```
Comment (vrajat) [inline rubix-presto/src/main/java/com/qubole/rubix/presto/CachingPrestoS3FileSystem.java]: I am surprised that this hasnt caused problems before.

Comment (abhishekdas99) [reply rubix-presto/src/main/java/com/qubole/rubix/presto/CachingPrestoS3FileSystem.java]: This is a recent change and has not been released :)

Comment (abhishekdas99) [reply rubix-presto/src/main/java/com/qubole/rubix/presto/CachingPrestoS3FileSystem.java]: Tried writing unit tests but its hard as we need to initialize the PrestoS3FileSystem

Comment (vrajat) [reply rubix-presto/src/main/java/com/qubole/rubix/presto/CachingPrestoS3FileSystem.java]: Too much mocking ? I am wondering if you can check that the base class is consistent at least.

Comment (abhishekdas99) [reply rubix-presto/src/main/java/com/qubole/rubix/presto/CachingPrestoS3FileSystem.java]: Tried mocking the class but the defeats the purpose checking scenarios like this. Have to figure out how to run these kind of check quickly.

**Pull request merged**